### ISSUE TITLE
报错内容ValueError: The provided lr scheduler "<torch.optim.lr_scheduler.LambdaLR object at 0x7fe3d759bb50>" is invalid

### ISSUE THREAD

**Issue by kkkjjjj1** [RetroCirce/HTS-Audio-Transformer#48]
按照教程在esc-50上进行训练，显示上面的报错是什么原因，怎么修改呢

**Comment by zszheng147** [comment]
Hi, bro, same problem here. Do you solve it?

**Comment by mqmmwu** [comment]
Hi, bro, same problem here. Do you solve it?

**Comment by RetroCirce** [comment]
Hi, I think the reason is probably due to the update of torch and pytorch lightning.
The code is implemented using torch 1.7.1 and pytorch lightning 1.5.9 --> but now they both have 2.x as latest version. 
Perhaps you can check if the declaration and assignment of the Lambda scheduler needs to be changed.

**Comment by ben100118** [comment]
是pytorch版本问题，你试试torch==1.10.0
